From f7819cc545a23845b2eeb9f46001e5003a6a34ce Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 13 Apr 2023 10:28:55 +0200
Subject: [PATCH 0166/1141] arm64: dts: st: add spi3 and spi8 on stm32mp257f-ev
 board

Add nodes for SPI3 and SPI8 on stm32mp257f-ev board.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I195b2d9ffee28ef994017efc9fc0b4c726ca2d4f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/301589
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev.dts | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
@@ -197,6 +197,20 @@
 	status = "okay";
 };
 
+&spi3 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&spi3_pins_a>;
+	pinctrl-1 = <&spi3_sleep_pins_a>;
+	status = "disabled";
+};
+
+&spi8 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&spi8_pins_a>;
+	pinctrl-1 = <&spi8_sleep_pins_a>;
+	status = "disabled";
+};
+
 &switch0 {
 	status = "okay";
 	pinctrl-0 = <&eth1_rgmii_pins_a>, <&eth3_rgmii_pins_a>;
